docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-landing-raw-html/__tests__/cgdp-landing-raw-html-cols.dom.ts

```diff
@@ -0,0 +1,155 @@
+export const cgdpRawHTMLRowCols = `
+<div class="usa-section usa-section--light" data-eddl-landing-row>
+    <div class="grid-container">
+        <div class="grid-row grid-gap-4">
+            <div class="tablet-lg:grid-col-8 ncids-column" data-eddl-landing-row-col>
+                <div class="ncids_dynamic_list" data-eddl-landing-item="collection">
```

Comment:
This test dom needs to get updated to have:
1 raw html in column 1, and 2 raw html in column 2, and test cases to match.

There should be matching click handlers in the unit tests.